sdk/analytics/src/main/java/com/klaviyo/analytics/networking/KlaviyoApiClient.kt

```diff
@@ -34,53 +34,20 @@ internal object KlaviyoApiClient : ApiClient {
      */
     private var apiObservers = mutableListOf<ApiObserver>()
 
-    init {
```

Comment:
The critical part of this PR: an exception inside an `init` is re-thrown, and besides the initializer doesn't finish properly and can't be re-invoked for a singleton object. Instead, I'm moving these listener startups to a method that we invoke from `Klaviyo.initialize`